edsdme/scripts/utils.js

```diff
@@ -469,8 +469,9 @@ export function getNodesByXPath(query, context = document) {
 }
 
 export function enableGeoPopup() {
-  const { hostname } = window.location;
-  if (hostname.endsWith('.adobe.com') && !partnerIsSignedIn()) {
+  const { hostname, search } = window.location;
+  const enableWithParam = new URLSearchParams(search).get('georouting') === 'on';
+  if ((hostname.endsWith('.adobe.com') && !partnerIsSignedIn()) || enableWithParam) {
```

Comment:
👏

Reply:
Milo only allows to use `georouting=off` to disable it and there is no way to enable it with `georouting=on`. We disable georouting if we are not on `.adobe.com` (stage or prod) and I added this so that we can easily enable it while developing or testing on feature branches.